kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -10819,6 +10829,120 @@ def find_mdev_device(vm_uuid, mdev_uuid):
+        try:
+            xml, mem_path, shmem_name = self._build_dgpu_shmem_xml(cmd.shmem)
+            vm = get_vm_by_uuid_no_retry(cmd.vmUuid, exception_if_not_existing=False)
+            if not vm or vm.state == Vm.VM_STATE_SHUTDOWN:
+                logger.debug("vm[uuid:%s] is absent or shutdown, no need to detach dGPU shmem[%s]" %
+                             (cmd.vmUuid, shmem_name))
+                linux.rm_file_force(mem_path)
+                return jsonobject.dumps(rsp)
+
+            try:
+                vm._wait_vm_run_until_seconds(60)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)